_posts/2015-10-08-design-is-multi-objective.md

```diff
@@ -4,20 +4,20 @@ title: Design is Multi-Objective
 ---
 
 The hardest thing (in optimization) is figuring out what you want.
-Even if we assume that all your wants are quantifiable, they’re probably not modelable: that is to say,
-even if you can assign an objective number to how good a design is (e.g., yearly profit of an airplane model),
-you probably can’t derive it from simpler quantities (e.g., that model’s range and speed).
+Even if we assume that all wants are quantifiable, they’re probably not modelable: that is to say,
+even if you can assign an objective number to how good a design is (e.g., yearly operating profit of an airplane model),
+it's not easy to derive that performance indicator from simpler quantities (e.g., that model’s range and speed).
 
 If you can do both, congratulations!
-What you want is so mathematical that, to you, one design is always better, worse, or interchangeable with another. But for most problems a single want (profit) is too much to model,
-and so we’re left with too many wants (speed, range) as proxies.
+Your objective is so mathematical that, to you, one design is always better, worse, or interchangeable with another. But for most problems a single want (like profit) is too difficult to model,
+and so we’re left with multiple wants (speed, range) as proxies.
 
 We need to explore a design space of competing alternatives, many of which have irreducable tradeoffs
 (e.g., between a fast short-hop plane and a slow long-distance one).
 The good news is that numerical optimization can take our complicated model straight to these tradeoffs;
 the bad is that even after that you still have to figure out which you want.
 
-Numerical optimization is often thought of as a way to “get an answer”.
+Numerical optimization is often thought of as a way to "get an answer".
 From a mathematical perspective, this is true;
 it’s like going to the Oracle with your linear program and asking the priests for the minimum possible value of $x$;
 after some deliberation, you get an answer you believe to be true: $3$.
```

Comment:
The Oracle analogy is a little confusing / hard to follow. Try to clean up for readers that might not be familiar with the references you are making.